installer/testcases/cpp/opencv_test/CMAKELists.txt

```diff
@@ -9,12 +9,16 @@ set(SENSING_DEV_DIR $ENV{SENSING_DEV_ROOT})
 
 # aravis
 include_directories(${SENSING_DEV_DIR}/include)
+# opencv
+include_directories(${SENSING_DEV_DIR}/opencv/build/include)
```

Comment:
Use `include_directories(${OpenCV_INCLUDE_DIRS})` instead of `include_directories(${SENSING_DEV_DIR}/opencv/build/include)` with `find_package`

set(SENSING_DEV_DIR $ENV{SENSING_DEV_ROOT})
if(NOT OpenCV_DIR)
    set(OpenCV_DIR $ENV{SENSING_DEV_ROOT}/opencv/build)
endif()

find_package(OpenCV REQUIRED PATHS ${OpenCV_DIR})